sqldef.go

```diff
@@ -47,9 +49,18 @@ func Run(generatorMode schema.GeneratorMode, db database.Database, sqlParser dat
 		return
 	}
 
-	sql, err := ReadFile(options.DesiredFile)
-	if err != nil {
-		log.Fatalf("Failed to read '%s': %s", options.DesiredFile, err)
+	var sql string
+	if options.FileSystem == nil {
+		sql, err = ReadFile(options.DesiredFile)
+		if err != nil {
+			log.Fatalf("Failed to read '%s': %s", options.DesiredFile, err)
+		}
+	} else {
+		data, err := fs.ReadFile(options.FileSystem, options.DesiredFile)
```

Comment:
Could it be possibly simpler to always use `fs.ReadFile` by providing something in the first argument when `options.FileSystem == nil`? I'm not sure what kind of `fs.ReadFile(x, y)` works like `ReadFile(y)`, but it might be nice if we can avoid complicating the implementation by this if-else branch.